protocol/avdl/crypto.avdl

```diff
@@ -45,6 +45,7 @@ protocol crypto {
     KID kid;
     EncryptedBytes32 ciphertext;
     BoxNonce nonce;
+    BoxPublicKey ePublicKey;
```

Comment:
I think publicKey is fine.  The service doesn't really care that it is ephemeral.